web3deployment/reserveDeploy/fpr/enhancedStepsDeployer.js

```diff
@@ -0,0 +1,76 @@
+const KyberReserveHighRate = artifacts.require("KyberReserveHighRate.sol");
+const ConversionRateEnhancedSteps = artifacts.require("ConversionRateEnhancedSteps.sol");
+const WrapConversionRateEnhancedSteps = artifacts.require("WrapConversionRateEnhancedSteps.sol");
+
+const BN = web3.utils.BN;
+
+let reserve;
+let reserveAddr = "";
+let conversionRate;
+let conversionRateAddr = "";
+let wrapper;
+let wrapperAddr = "";
+
+let networkAddr = "0x920b322d4b8bab34fb6233646f5c87f87e79952b";
+let admin= "0xf3D872b9E8d314820dc8E99DAfBe1A3FeEDc27D5";
+let deployer;
+
+async function main() {
+    const accounts = await web3.eth.getAccounts();
+    deployer = accounts[0];
+    console.log(`deployer address at ${deployer}`);
+
+    gasPrice = new BN(75).mul(new BN(10).pow(new BN(9)))
```

Comment:
maybe use yargs for gas price?

Reply:
if we run the script with buidler, we can not read value from args
but we can read the value from environment variable